artifacts/CAMARA_common.yaml

```
@@ -370,6 +370,14 @@ components:
                 status: 422
                 code: DEVICE_NOT_APPLICABLE
                 message: The service is not available for the provided device.
+            GENERIC_422_UNIDENTIFIABLE_DEVICE:
+              description: The device identifier is not included in the request and the device information cannot be derived from the 3-legged access token
+
+              value:
+                status: 422
+                code: UNIDENTIFIABLE_DEVICE
+
+                message: The device cannot be identified.

```

Delete the two empty lines
```suggestion
            GENERIC_422_UNIDENTIFIABLE_DEVICE:
              description: The device identifier is not included in the request and the device information cannot be derived from the 3-legged access token
              value:
                status: 422
                code: UNIDENTIFIABLE_DEVICE
                message: The device cannot be identified.
```